fk/secretsend.go

```diff
@@ -126,6 +126,13 @@ func getSecretFromFile(filename string, fileReader ioutilReadFileInterface, prom
 	if len(strings.TrimSpace(secret)) == 0 {
 		return "", fmt.Errorf(filename + " is empty")
 	}
+	if !utf8.ValidString(secret) {
```

Comment:
Ooh GOOD FIND!

Reply:
I *suspect* that `utf8.ValidString` probably covers all of our disallowed rune cases... but never mind!

Reply:
You're right, it can be utf8 *and* have disallowed runes